doc/method/method-details.md

```diff
@@ -21,22 +19,27 @@ are performed:
     a way to say something about the members.  The fact that a node
     had originally been in a container is recorded as a flag on the
     child node.
- 1. Monotypic homonym removal - when node with name N has as its
+ 1. Monotypic homonym removal - when a node with name N has as its
     only child another node with the same name N, the parent is removed.
     This is done to avoid ambiguities when aligning taxonomies.
     [get examples by rerunning]
+ 1. Diacritics removal - accents and umlauts are removed in order to improve 
+    name matching, as well as to follow the nomenclatural codes, which prohibit them.
+    The original name-string is kept, as a synonym.
 
-[KC: need to say something about whether these cases get touched again during
-the process, i.e. do these nodes ever get added back, or are they permanently
-removed?]
+[KC: need to say something about whether these cases get touched again
+during the process, i.e. do these nodes ever get added back, or are
+they permanently removed?  JAR: something like "the normalized
+versions of the taxonomies then become the input to subsequent
+processing phases"?]
```

Comment:
but still confused about what needs to be said here. I don't think it would be logical to ask whether a maybe node gets added back, since it clearly says the parent is removed. It would be surprising if it *were* added back. I made a suggestion, what do you think should be said?